src/utils/utils.js

```diff
@@ -6,13 +6,34 @@ const mainApi = axios.create({
 
 function getReviews (){
     return mainApi
-    .get('./reviews')
+    .get(`./reviews`)
     .then((res) =>{
         return res.data.reviews;
     })
     .catch((err) =>{
         console.log(err);
     })
 }
+function getReviewById (id){
+    return mainApi
+    .get(`./reviews/${id}`)
+    .then((res) =>{
+        return res.data;
+    })
+    .catch((err) =>{
+        console.log(err);
+    })
+}
+
+function getComments (id){
+    return mainApi
+    .get(`./reviews/${id}/comments`)
+    .then((res) =>{
+        return res.data.comments;
```

Comment:
When I go to a review with no comments, the page errors with a 404 error from Axis. Does your backend respond with a 404 if there are no comments on a valid review? If So, this should be a 200 with an empty array